app/config.py

```diff
@@ -100,8 +100,6 @@ class Config(object):
     # URL of redis instance
     REDIS_URL = os.getenv('REDIS_URL')
     REDIS_ENABLED = os.getenv('REDIS_ENABLED') == 'True'
-    API_RATE_LIMIT_ENABLED = os.getenv('API_RATE_LIMIT_ENABLED', 'False') == 'True'
-    API_MESSAGE_LIMIT_ENABLED = os.getenv('API_MESSAGE_LIMIT_ENABLED', 'False') == 'True'
```

Comment:
`API_RATE_LIMIT_ENABLED` and `API_MESSAGE_LIMIT_ENABLED` need to stay. They are used in the dev & staging, but default to `False` in perf & prod.